alacritty_terminal/src/event_loop.rs

```diff
@@ -213,8 +214,9 @@ where
             let mut interest = PollingEvent::readable(0);
 
             // Register TTY through EventedRW interface.
-            unsafe {
-                self.pty.register(&self.poll, interest, poll_opts).unwrap();
+            if let Err(err) = unsafe { self.pty.register(&self.poll, interest, poll_opts) } {
+                error!("EventLoop register error: {err:?}");
```

Comment:
Errors are generally better logged with their `Display` impl. Also think this reads a little nicer as "registration error" rather than "register error", since it's not an error with a register (like a CPU register).

Should also be `Event loop`, rather than `EventLoop`.